ReeldIn/settings.py

```diff
@@ -34,7 +34,16 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True  # TODO update via env variable
 
-ALLOWED_HOSTS = ["reeld.in", "www.reeld.in", "127.0.0.1"]
+ALLOWED_HOSTS = [
+    "reeld.in",
+    "www.reeld.in",
+    "localhost",
+    "reeldin.azurewebsites.net",
+    "169.254.129.3",
```

Comment:
Might want to remove this in the final? Assuming its your IP.

Reply:
No thats the webapp ip. I got disallowed host when it wasn't there